### PR TITLE
Reconcile the exact same resources as the old operator

### PR DESCRIPTION
**Pull request by shwetaap**
Signed-off-by: Shweta Padubidri <spadubid@redhat.com>

**What this PR does / why we need it**:
Reconcile the exact same resources as the old operator to provide zero interruption upgrade

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
None
```

### CODE REVIEW

File: internal/operands/template-validator/resources.go [hunk 1]
```diff
@@ -16,38 +16,30 @@ import (
...
-	return virtTemplateValidator + "-" + namespace
-}
-
 func newClusterRole(namespace string) *rbac.ClusterRole {
 	return &rbac.ClusterRole{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   ClusterRoleName(namespace),
-			Labels: commonLabels,
+			Name:      ClusterRoleName,
+			Namespace: namespace,
```
Comment (akrejcir) [inline internal/operands/template-validator/resources.go:39]: This is a cluster role, so the namespace needs to be `""`.

File: internal/operands/template-validator/resources.go [hunk 2]
```diff
@@ -70,12 +62,13 @@ func newServiceAccount(namespace string) *core.ServiceAccount {
 func newClusterRoleBinding(namespace string) *rbac.ClusterRoleBinding {
 	return &rbac.ClusterRoleBinding{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:   ClusterRoleBindingName(namespace),
-			Labels: commonLabels,
+			Name:      ClusterRoleBindingName,
+			Namespace: namespace,
```
Comment (akrejcir) [inline internal/operands/template-validator/resources.go:66]: The same here. The namespace needs to be empty.

**Review by akrejcir — CHANGES_REQUESTED** [review]

File: internal/operands/template-validator/resources.go (hunk 1 above)
Comment (shwetaap) [reply internal/operands/template-validator/resources.go]: Done

File: internal/operands/template-validator/resources.go (hunk 2 above)
Comment (shwetaap) [reply internal/operands/template-validator/resources.go]: Done

**Pull request merged**